scripts/qa_crowbarsetup.sh

```diff
@@ -3566,10 +3566,13 @@ function oncontroller_manila_generic_driver_setup()
 function oncontroller_magnum_service_setup ()
 {
     # (mmnelemane): Replace this Fedora image with a suitable SLES image when available
-    local service_image_name=magnum-service-image.qcow2
-    local service_image_url=http://clouddata.cloud.suse.de/images/$(uname -m)/other/$service_image_name
+    #ocal service_image_name=magnum-service-image.qcow2
+    #local service_image_url=http://clouddata.cloud.suse.de/images/$(uname -m)/other/$service_image_name
 
-    if ! openstack image list --f value -c Name | grep -q "^magnum-service-image$"; then
+    local service_image_name=SLE12SP1-JeOS-k8s-magnum.x86_64.qcow2
+    local service_image_url="http://download.suse.de/ibs/Devel:/Docker:/Images:/SLE12SP1-JeOS-k8s-magnum/images/$service_image_name"
```

Comment:
not sure that we want to point directly to the images. it is imho better to point to a known good image, and have a job that updates that one only with a new image when the new image is known to work